readthedocs/search/tests/test_api.py

```diff
@@ -526,7 +526,7 @@ def test_search_advanced_query_detection(self, api_client):
         search_params = {
             "project": project.slug,
             "version": version.slug,
-            "q": "indx",
+            "q": "index",
```

Comment:
This was probably an invalid word on purpose.

Reply:
so we need to annotate this line to be skipped and ok to contain `indx`?

Reply:
I replaced it with explicit `"abracadabra"` , or it does have to be `"indx"`? (let's see if tests feel better)